### PR TITLE
Add clamp and split

### PR DESCRIPTION
**Pull request by huningxin**
fix https://github.com/webmachinelearning/webnn/issues/66#issuecomment-699670404

@wchao1115 @pyu10055 @gramalingam @anssiko , PTAL. Thanks!

***
<a href="https://pr-preview.s3.amazonaws.com/huningxin/webnn/pull/95.html" title="Last updated on Oct 9, 2020, 3:00 AM UTC (8c18b5c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/95/65772f1...huningxin:8c18b5c.html" title="Last updated on Oct 9, 2020, 3:00 AM UTC (8c18b5c)">Diff</a>

### CODE REVIEW

**Review by anssiko — APPROVED** [review]

**Comment by huningxin** [comment]
Added `split` that would fix #66 , please take another look. Thanks!

**Review by wchao1115 — APPROVED** [review]

**Review by anssiko — APPROVED** [review]

**Comment by huningxin** [comment]
Fixed the conflicts after #94 merged. I am going to merge this PR if there are no other comments.

**Comment by huningxin** [comment]
With the two approvals, I am merging this PR. As this PR also fix the bikeshield build issue, It will also unblock the #101 .

**Pull request merged**
